extensions/data-plane/data-plane-http/src/main/java/org/eclipse/edc/connector/dataplane/http/pipeline/HttpDataSourceFactory.java

```diff
@@ -52,12 +52,21 @@ public boolean canHandle(DataFlowRequest request) {
 
     @Override
     public @NotNull Result<Boolean> validate(DataFlowRequest request) {
+        if (validateRequest(request).succeeded()) {
+            return VALID;
+        } else {
+            return Result.failure("The validation failed");
+        }
```

Comment:
I think this could be simplified to:
`return validateRequest(request).map(it -> true);`